stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Page.java

```diff
@@ -147,6 +150,22 @@ public void clearDeviceOrientationOverride(JsonRpcPeer peer, JSONObject params)
   }
 
   @ChromeDevtoolsMethod
+  public void startScreencast(final JsonRpcPeer peer, JSONObject params) {
+    final StartScreencastRequest request = mObjectMapper.convertValue(
+        params, StartScreencastRequest.class);
+    mScreencastDispatcher.startScreencast(peer, request);
+  }
+
+  @ChromeDevtoolsMethod
+  public void stopScreencast(JsonRpcPeer peer, JSONObject params) {
+    mScreencastDispatcher.stopScreencast();
+  }
+
+  @ChromeDevtoolsMethod
+  public void screencastFrameAck(JsonRpcPeer peer, JSONObject params) {
+  }
```

Comment:
since this is empty, it can be help to add a comment like this so that future readers don't think this is accidental, e.g.
```
// Nothing to do here, just need to make sure Chrome doesn't get an error that this method isn't implemented
```